src/server/AppManager.ts

```diff
@@ -765,4 +783,32 @@ export class AppManager {
 
         return enable;
     }
+
+    private createAppUser(app: ProxiedApp): Promise<string> {
+        const userData: Partial<IUser> = {
+            username: app.getInfo().nameSlug,
+            name: app.getInfo().name,
+            roles: ['app'],
+            appId: app.getID(),
+            type: UserType.APP,
+            status: 'online',
+            isEnabled: true,
+        };
+
+        return this.bridges.getUserBridge().create(userData, app.getID(), {
+            avatarUrl: app.getInfo().iconFileContent || app.getInfo().iconFile,
+            joinDefaultChannels: true,
```

Comment:
I don't think app users should join the default channels.

Reply:
As discussed offline, we decided to keep the `joinDefaultChannels` as `true` as it is a way for users to find out that a new app has been installed